main.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# bubble sort
+def buble(arr)
+  n_times = arr.length - 1
+  n_times.times do |i|
+    arr[i], arr[i + 1] = arr[i + 1], arr[i] if arr[i] > arr[i + 1]
+  end
+  print arr
+end
```

Comment:
- [ ] This method is not passing the tests, if I try putting: `p buble([6,5,4,3,2,1])` it gives: `[5, 4, 3, 2, 1, 6]`. **Hint:** Your logic is basically ok, but you are only sorting it one time, that is why it only sorts the 6.  You need to give as many passes as necessary until everything is sorted